projects/python/perception/object_detection_2d/nanodet/inference_demo.py

```diff
@@ -22,10 +22,12 @@
     parser = argparse.ArgumentParser()
     parser.add_argument("--device", help="Device to use (cpu, cuda)", type=str, default="cuda", choices=["cuda", "cpu"])
     parser.add_argument("--model", help="Model for which a config file will be used", type=str, default="m")
+    parser.add_argument("--dynamic", help="Determines if model run with dynamic shape input or not",
```

Comment:
This is wrong from my part. It is true that it can be used in Nanodet Plus head with legacy_post_process=False if the model is not optimized with any framework but it changes how optimizations are working. I will change what it is needed across script descriptions and I will add that. Dynamic input results to slower inference times.

Reply:
Oh ok, then this probably indeed needs fixing on several places that it is mentioned. Feel free to fix them as you think it's better.